inst/pages/quality_control.qmd

```diff
@@ -256,6 +256,25 @@ top_features <- getTop(tse, method="median", top=10)
 rowData(tse)[top_features, taxonomyRanks(tse)][1:5, 1:3]
 ```
 
+### Unique taxa
+
+The function `getUnique` returns unique taxa in the data.
+
+```{r}
+# Get unique taxa at rank Class
+getUnique(tse, "Class", sort = TRUE)
+```
+
+### Summarizing data
+
+Use the function `summary` to get the summary of counts for all samples and 
+features in SummarizedExperiment object.
+
+```{r}
+# Get an overview of sample and taxa counts
+summary(tse, assay_name= "counts")
```

Comment:
isn't this assay.type nowadays (assay_name should be deprecated?)

Reply:
there is also some single cell summary function available, could be mentioned on the same go?

Reply:
We have [scuttle::addPerCellQC](https://rdrr.io/pkg/scuttle/man/addPerCellQCMetrics.html), also mentioned in OMA https://microbiome.github.io/OMA/docs/devel/pages/quality_control.html - this can also provide some summaries, would be useful to show both in the same place (you can see what place works best)?